lampod-cli/src/main.rs

```diff
@@ -44,15 +48,57 @@ fn main() -> error::Result<()> {
     Ok(())
 }
 
+fn write_words_to_file<P: AsRef<Path>>(path: P, words: &Option<String>) -> io::Result<()> {
+    let mut file = OpenOptions::new()
+        .write(true)
+        .create(true)
+        .truncate(true)
+        .open(path)?;
+
+    match words {
+        Some(ref value) => {
+            file.write_all(value.as_bytes())?;
+        }
+        None => {
+            file.write_all(b"")?;
+        }
+    }
+
+    Ok(())
+}
+
+fn load_words_from_file<P: AsRef<Path>>(path: P) -> io::Result<Option<String>> {
+    let mut file = File::open(path)?;
+    let mut content = String::new();
+
+    file.read_to_string(&mut content)?;
+
+    if content.is_empty() {
+        Ok(None)
+    } else {
+        Ok(Some(content))
+    }
+}
+
 /// Return the root directory.
 fn run(args: LampoCliArgs) -> error::Result<()> {
+    let path = std::env::home_dir().expect("Impossible to get the home directory");
+    let path = path.to_str().unwrap();
+    let words_path = format!("{}/.lampo/{}", path, args.network.as_ref().unwrap());
```

Comment:
we may have the lampo home that include the network somewhere else? probably the LampoConf? or similar?

Reply:
There is a directory in lampo conf but we might have to clone the `args` as `args` are getting borrowed while creating lampo conf. If we generate lampo conf before we get error in this line.
`let mnemonic = if args.restore_wallet {`

Reply:
You can take the reference of the path before the `let-if`?  If you provide an example I could help you to fix this problem

Reply:
I have made a fix. It should be fine now.